intro-basics-2023-1/Assets/Scripts/EnemyTank.cs

```diff
@@ -0,0 +1,79 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyTank : MonoBehaviour
+{
+    [SerializeField]
+    private float _speed = 1;
+    
+    [SerializeField]
+    private Transform _playetTank;
+
+    [Space(20)]
+    [SerializeField]
+    private Transform[] _cannons;
+    
+    [SerializeField]
+    private GameObject _projectilePrefab;
+
+    [SerializeField]
+    private Transform[] _shootPoints;
+
+    private Rigidbody2D _rb;
+
+    private float _sentido; 
+    
+    private float _timeToShot = 3;
+
+    private void Start()
+    {
+        gameObject.SetActive(false);
+        _rb = GetComponent<Rigidbody2D>();
+        _sentido = 1;
+    }
+    
+    void Update()
+    {
+        //Cannon Rotation
+        Vector3 aimVector = (_playetTank.position - _cannons[0].position).normalized;
```

Comment:
La ventaja de tener un array `[]` es que puedes utilizar un `for` para escribir una sola vez el codigo y aplicarlo a todos los elementos.